src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -46,33 +46,33 @@ typedef void (*EnHorseCsFunc)(struct EnHorse*, PlayState*, CsCmdActorAction*);
 #define ENHORSE_FLAG_30 (1 << 30)              /*  0x40000000 */
 #define ENHORSE_FLAG_31 (1 << 31)              /*  0x80000000 */
 
-typedef enum {
-    /* 0  */ ENHORSE_ACT_FROZEN,
-    /* 1  */ ENHORSE_ACT_INACTIVE,
-    /* 2  */ ENHORSE_ACT_IDLE,
-    /* 3  */ ENHORSE_ACT_FOLLOW_PLAYER,
-    /* 4  */ ENHORSE_ACT_INGO_RACE,
-    /* 5  */ ENHORSE_ACT_MOUNTED_IDLE,
-    /* 6  */ ENHORSE_ACT_MOUNTED_IDLE_WHINNEYING,
-    /* 7  */ ENHORSE_ACT_MOUNTED_TURN,
-    /* 8  */ ENHORSE_ACT_MOUNTED_WALK,
-    /* 9  */ ENHORSE_ACT_MOUNTED_TROT,
-    /* 10 */ ENHORSE_ACT_MOUNTED_GALLOP,
-    /* 11 */ ENHORSE_ACT_MOUNTED_REARING,
-    /* 12 */ ENHORSE_ACT_STOPPING,
-    /* 13 */ ENHORSE_ACT_REVERSE,
-    /* 14 */ ENHORSE_ACT_LOW_JUMP,
-    /* 15 */ ENHORSE_ACT_HIGH_JUMP,
-    /* 16 */ ENHORSE_ACT_BRIDGE_JUMP,
-    /* 17 */ ENHORSE_ACT_CS_UPDATE,
-    /* 18 */ ENHORSE_ACT_HBA,
-    /* 19 */ ENHORSE_ACT_FLEE_PLAYER,
-    /* 20 */ ENHORSE_ACT_20,
+typedef enum EnHorseAction {
+    /*  0 */ ENHORSE_ACT_FROZEN,
+    /*  1 */ ENHORSE_ACT_INACTIVE,
+    /*  2 */ ENHORSE_ACT_IDLE,
+    /*  3 */ ENHORSE_ACT_FOLLOW_PLAYER,
+    /*  4 */ ENHORSE_ACT_INGO_RACE,
+    /*  5 */ ENHORSE_ACT_5,
+    /*  6 */ ENHORSE_ACT_6,
+    /*  7 */ ENHORSE_ACT_MOUNTED_IDLE,
+    /*  8 */ ENHORSE_ACT_MOUNTED_IDLE_WHINNEYING,
+    /*  9 */ ENHORSE_ACT_MOUNTED_TURN,
+    /* 10 */ ENHORSE_ACT_MOUNTED_WALK,
+    /* 11 */ ENHORSE_ACT_MOUNTED_TROT,
+    /* 12 */ ENHORSE_ACT_MOUNTED_GALLOP,
+    /* 13 */ ENHORSE_ACT_MOUNTED_REARING,
+    /* 14 */ ENHORSE_ACT_STOPPING,
+    /* 15 */ ENHORSE_ACT_REVERSE,
+    /* 16 */ ENHORSE_ACT_LOW_JUMP,
+    /* 17 */ ENHORSE_ACT_HIGH_JUMP,
+    /* 18 */ ENHORSE_ACT_CS_UPDATE,
+    /* 19 */ ENHORSE_ACT_HBA,
+    /* 20 */ ENHORSE_ACT_FLEE_PLAYER,
     /* 21 */ ENHORSE_ACT_21,
     /* 22 */ ENHORSE_ACT_22,
     /* 23 */ ENHORSE_ACT_23,
     /* 24 */ ENHORSE_ACT_24,
-    /* 25 */ ENHORSE_ACT_25,
+    /* 25 */ ENHORSE_ACT_25
```

Comment:
I'd prefer not to abbreviate `ACTION`, it's only 3 more letters, and we might want `ACT` for something else.